include/translation.h

```diff
@@ -13,4 +13,7 @@
  */
 #define T(jp, en) jp
 
+// Note: need to also include debug.h if using this macro
+#define LOG_STRING_T(stringJP, stringEN, file, line) LOG(#stringJP, stringJP, "%s", file, line)
+
```

Comment:
I would just include debug.h all the time at this point.
Even if its only used for a smaller subset of cases, imo its still better than needing a header for another header